.travis.yml

```
@@ -79,7 +79,9 @@ before_install:
   - export PERL5LIB=/usr/share/perl5
 
     # Provide cpanm helper
-  - eval $(curl https://travis-perl.github.io/init) --auto
+    # the eval call combines all lines into one, therefore comments should be
+    # removed before executing the script
+  - eval $(curl https://travis-perl.github.io/init | grep -v '#') --auto
```

Couldn't we just quote it?

```suggestion
  - eval "$(curl https://travis-perl.github.io/init)" --auto
```

---

you just killed the game ;)